.github/workflows/rust.yml

```diff
@@ -61,11 +61,34 @@ jobs:
           command: test
           args: --all-features --no-fail-fast
 
-  test-integration-ignored:
+  # TODO(shonfeder): remove duplication once GitHub addresses
+  #  - https://github.community/t/support-for-yaml-anchors/16128/15, or
+  #  - https://github.community/t/reusing-sharing-inheriting-steps-between-jobs-declarations/16851/13
+  test-integration-stable:
     runs-on: ubuntu-latest
     services:
       tendermint:
-        image: tendermint/tendermint
+        image: tendermint/tendermint:v0.33.5
```

Comment:
It would be nice to set this version somewhere more canonical, but from what I've seen of github actions so far, there's no clear way to parameterize the values of an `image` field here. I'd be interested in suggestions for a more elegant approach here. Failing that, I could at least document this somewhere (and the question is just where that should go).

Reply:
I don't have much experience with gh either. I think even having the version more prominently in an env variable, might be ok: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables

@marbar3778 as you looked a lot more into GA, do you have a suggestion how to parameterize this?